ur_robot_driver/scripts/tool_communication

```diff
@@ -1,6 +1,23 @@
 #!/usr/bin/env python
 """Small helper script to start the tool communication interface"""
 
+# -- BEGIN LICENSE BLOCK ----------------------------------------------
```

Comment:
@fmauch Should we add this in another PR instead?  It does not seem directly related to this PR.

Reply:
yes, that might be cleaner...

Reply:
done in current version.